Symfony/CS/Tokenizer/Tokens.php

```diff
@@ -670,6 +778,12 @@ public function removeTrailingWhitespace($index, array $opts = array())
      */
     public function setCode($code)
     {
+        // No need to work when the code is the same.
+        // That is how we avoid a lot of work and setting changed flag.
+        if ($code === $this->generateCode()) {
+            return;
+        }
```

Comment:
It isn't used on 1.x line...
Both 1.4 and 1.5 does not use `setCode` method. Even current master won't use it.

It may or may not change performance for some non-commited to repo contributors fixers, and we can't test it.
But if someone corrupt some tokens and have tokens like (_ for spaces):
`=___`, `1` and want to set same code only to reparse into `=`, `___`, `1`
we will change that behavior.
Couse on 1.x we wont care about having valid every single token after each fixer, after this PR we will care. So we can start doing that early return.